app/services/waste_carriers_engine/order_copy_cards_completion_service.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module WasteCarriersEngine
+  class OrderCopyCardsCompletionService < BaseService
+    include CanMergeFinanceDetails
+
+    attr_reader :transient_registration
+
+    delegate :registration, to: :transient_registration
+
+    def run(transient_registration)
+      @transient_registration = transient_registration
+
+      complete_order_copy_cards
+    end
+
+    private
+
+    def complete_order_copy_cards
+      create_past_registration
```

Comment:
I'd want @irisfaraway to confirm, but I don't think we need this or `create_past_registration()`. If I've read `PastRegistration. build_past_registration` correctly, in this situation it is always going to return `null` and not create anything so there is no need to call it.

Plus I don't believe we should be storing a copy of the registration in past registrations just because a copy card has been ordered.

Reply:
Agreed, this only exists if the registration is being modified. All we're doing is creating new orders and payments, which is dealt with differently than the other data.